guides/releasing/Release Workflow.md

```diff
@@ -481,7 +481,6 @@ Once everything above is completed and Apple has approved the iOS client, the fi
 
 - Publish a new version of `turtle-cli` [following this guide](https://github.com/expo/turtle/blob/master/CONTRIBUTING.md#publishing-a-release).
 - Follow the instructions in the [`turtle-deploy` README](https://github.com/expo/turtle-deploy/). (Note that it refers to CI jobs in the `turtle` repo, not its own repo.)
-- Promote the `expokit` package published in step 5.1 to `latest` on NPM.
```

Comment:
Shouldn't we also update `5.1.` section?

Reply:
Fixed in https://github.com/expo/expo/pull/9982/commits/2f8aadfde61567546cae9b2a7eba4a70a7051f6f